ocrd_cis/ocropy/clip.py

```diff
@@ -87,8 +87,8 @@ def process(self):
             
             page_image, page_coords, page_image_info = self.workspace.image_from_page(
                 page, page_id, feature_selector='binarized')
-            zoom, dpi = determine_zoom(self.parameter['dpi'], page_image_info)
-            self.logger.info(f"Page '{page_id}' uses {dpi} DPI. Determined zoom={zoom}")
+            # TODO: zoom is not used anywhere, is it still useful to have this call here?
```

Comment:
Not sure at this point. The methods I used here are somewhat independent of ocrolib, but I have not tested them systematically on various DPIs.

Reply:
The only difference if I remove that line would be the missing info log.

Reply:
oh, I was merely referring to the point you raised with that todo.

Reply:
Ah, gotcha. 

I would actually prefer to keep the `determine_zoom` in, but then add the comment as well.